### PR TITLE
feat: add send_voice_messages permission

### PR DESCRIPTION
**Pull request by onerandomusername**
## Summary

Adds the new permission for sending voice messages.

## Checklist

- [x] If code changes were made, then they have been tested
    - [x] I have updated the documentation to reflect the changes
    - [x] I have formatted the code properly by running `pdm lint`
    - [x] I have type-checked the code by running `pdm pyright`
- [ ] This PR fixes an issue
- [x] This PR adds something new (e.g. new method or parameters)
- [ ] This PR is a breaking change (e.g. methods or parameters removed/renamed)
- [ ] This PR is **not** a code change (e.g. documentation, README, ...)

### CODE REVIEW

**Comment by shiftinv** [comment]
This should be combined with the other voice message related changes, and have one changelog entry for the permission bit + other fields.
I've pushed the other things [here](https://github.com/DisnakeDev/disnake/compare/feature/voice-messages), would be great if you could pull them into this PR.

**Comment by onerandomusername** [comment]
> This should be combined with the other voice message related changes, and have one changelog entry for the permission bit + other fields. I've pushed the other things [here](https://github.com/DisnakeDev/disnake/compare/feature/voice-messages), would be great if you could pull them into this PR.

I'd rather get the permission in now, as that doesn't require much of a review.

**Comment by shiftinv** [comment]
> I'd rather get the permission in now, as that doesn't require much of a review.

Fine, I suppose.

**Review by shiftinv — CHANGES_REQUESTED** [review]
Not having `send_messages` permissions should implicitly deny `send_voice_messages`:
image

**Comment by onerandomusername** [comment]
> Not having `send_messages` permissions should implicitly deny `send_voice_messages`: image

Fair, I forgot about this because that is part of the code is completely wrong.

File: changelog/1006.feature.rst
Comment (shiftinv) [inline changelog/1006.feature.rst:1]: The old changelog entry should be updated to match as well

File: disnake/permissions.py [hunk 1]
```diff
@@ -408,6 +409,7 @@ def text(cls) -> Self:
             read_message_history=True,
             send_tts_messages=True,
             use_slash_commands=True,
+            send_voice_messages=True,
```
Comment (shiftinv) [inline disnake/permissions.py:412]: This is missing a `versionchanged`, same with `.all_channel()`

File: disnake/permissions.py [hunk 2]
```diff
@@ -314,6 +315,7 @@ def all_channel(cls) -> Self:
 
         .. versionchanged:: 2.9
             Added :attr:`use_soundboard` permission.
+            Added :att:`send_voice_messages` permission.
```
Comment (shiftinv) [inline disnake/permissions.py:318]:
```suggestion
            Added :attr:`use_soundboard` and :attr:`send_voice_messages` permissions.
```

File: disnake/permissions.py [hunk 3]
```diff
@@ -391,6 +393,9 @@ def text(cls) -> Self:
         .. versionchanged:: 2.0
            Added :attr:`create_public_threads`, :attr:`create_private_threads`, :attr:`manage_threads`,
            :attr:`send_messages_in_threads` and :attr:`use_external_stickers` permissions.
+
+        .. versionchanged:: 2.9
+            Added :att:`send_voice_messages` permission.
```
Comment (shiftinv) [inline disnake/permissions.py:398]:
```suggestion
            Added :attr:`send_voice_messages` permission.
```

**Review by shiftinv — APPROVED** [review]
lgtm, ty

**Pull request merged**